verifier/src/main/java/com/aws/jverify/verifier/compiler/frontend/LowerInterceptor.java

```diff
@@ -0,0 +1,71 @@
+package com.aws.jverify.verifier.compiler.frontend;
+
+import com.sun.tools.javac.code.Kinds;
+import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.comp.Lower;
+import com.sun.tools.javac.tree.JCTree;
+import com.sun.tools.javac.tree.TreeMaker;
+import com.sun.tools.javac.util.List;
+import net.bytebuddy.implementation.bind.annotation.*;
+
+import java.lang.reflect.Field;
+import java.util.concurrent.Callable;
+
+/*
+Contains several intercepting methods. 
+The lowercased suffix of the name of methods is used to determine what it is intercepting
+If the name does not match anything, then the method is used to intercept the remainder of what needs intercepting
+ */
```

Comment:
Teensy nit in general:

```suggestion
/*
 * Contains several intercepting methods. 
 * The lowercased suffix of the name of methods is used to determine what it is intercepting
 * If the name does not match anything, then the method is used to intercept the remainder of what needs
 * intercepting
 */
```

Although I've started using `///` markdown comments instead everywhere myself.

Reply:
What's the benefit of those?